frontend/javascripts/oxalis/default_state.ts

```diff
@@ -68,6 +68,7 @@ const defaultState: OxalisState = {
     centerNewNode: true,
     overrideNodeRadius: true,
     particleSize: 5,
+    presetBrushSizesAsc: [],
```

Comment:
I felt torn between creating an object or leaving the order in the name. I am only 50% happy with this solution, but creating an object {smallBrushSize, mediumBrushSize, largeBrushSize} seemed over the top. Maybe

Reply:
I also wasn't sure about "Asc" or "Ascending" but "Asc" is known because of SQL, so I kept it. Looking forward to your input! @philippotto

Reply:
alright! so I would simply create a type `BrushPresets` in the store?

Reply:
yes, that should work :+1: